client/src/utils/api.js

```diff
@@ -82,6 +82,16 @@ export async function getSingleEpisodeFromDB(id) {
   return singleEpisode;
 }
 
+export async function addOrUpdateEpisodeInDB(id, userId, liked) {
```

Comment:
A POST should create a new entity and not update an existing one, this breaks http semantics. PATCH and PUT are idempotent and more suited for updating.